### PR TITLE
Add `potatoland4492` subdomain

### PR DESCRIPTION
**Pull request by potatoland4492**

### CODE REVIEW

File: domains/potatoland4492.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "A personal site for all my projects.",
+  "repo": "https://github.com/potatoland4492/devpage",
+  "owner": {
+    "username": "potatoland4492",
+    "email": "potatoland4492@gmail.com"
+  },
+  "record": {
+    "CNAME": "potatoland4492.github.io/devpage"
```
Comment (MaskDuck) [inline domains/potatoland4492.json:9]: cname cant have slash

**Review by MaskDuck — DISMISSED** [review]

File: domains/potatoland4492.json (hunk 1 above)
Comment (potatoland4492) [reply domains/potatoland4492.json]: sorry bout that...
fixed

Comment (potatoland4492) [reply domains/potatoland4492.json]: Please merge, I fixed the CNAME record

**Comment by omxpro** [comment]
@MaskDuck changes fixed

**Review by TheBotlyNoob — DISMISSED** [review]
cool

**Review by TheBotlyNoob — APPROVED** [review]

**Pull request merged**